virtuoso.sh

```diff
@@ -58,7 +58,7 @@ then
     echo "WAIT_FOR_CHILDREN; " >> /load_data.sql
     echo "$(cat /load_data.sql)"
     virtuoso-t +wait && isql-v -U dba -P "$pwd" < /load_data.sql
-    kill $(ps aux | grep '[v]irtuoso-t' | awk '{print $2}')
+    kill "$(ps aux | grep '[v]irtuoso-t' | awk '{print $2}')"
```

Comment:
as previously discussed, quoting is not required here and may even result in unwanted behavior